shap/explainers/_tree.py

```diff
@@ -1659,7 +1659,10 @@ def __init__(self, tree, normalize=False, scaling=1.0, data=None, data_missing=N
         if safe_isinstance(tree, ["sklearn.tree._tree.Tree", "econml.tree._tree.Tree"]):
             self.children_left = tree.children_left.astype(np.int32)
             self.children_right = tree.children_right.astype(np.int32)
-            self.children_default = self.children_left  # missing values not supported in sklearn
+            if hasattr(tree, "missing_go_to_left"):
+                self.children_default = np.where(tree.missing_go_to_left, self.children_left, self.children_right)
+            else:
+                self.children_default = self.children_left
```

Comment:
We can get rid of the codecov warning if we do

```suggestion
           self.children_default = self.children_left
           if hasattr(tree, "missing_go_to_left"):
                self.children_default = np.where(tree.missing_go_to_left, self.children_left, self.children_right)
```

Reply:
fixed, thanks